tests/test_enterprise/api/test_throttles.py

```diff
@@ -35,14 +37,38 @@ def setUp(self):
 
         self.url = settings.TEST_SERVER + reverse('enterprise-customer-list')
 
+    def tearDown(self):
+        # This will clear throttle limits between tests
+        cache.clear()
+
     def exhaust_throttle_limit(self, throttle_limit):
         """
         Call enterprise api so that throttle limit is exhausted.
         """
-        for item in range(throttle_limit):  # pylint: disable=unused-variable
+        for _ in range(throttle_limit):
             response = self.client.get(self.url)
             assert response.status_code == status.HTTP_200_OK
 
+
+    @override_settings(USER_THROTTLE_RATE='10/minute', SERVICE_USER_THROTTLE_RATE='15/minute')
+    def _exhaust_service_worker_and_assert_429(self, username, password):
```

Comment:
I get the pattern here isn't new and does make sense, but I've seen issues with it being flakey in other repos in the past (credentials). Anyway, it's fine but just want to call that out

Reply:
Yup, I'm hoping that setting the throttle rates much lower on these tests not only speeds them up, but also makes them less likely to flake.